cli/src/commands/dbs/delete.ts

```diff
@@ -27,14 +27,15 @@ export default class DatabasesDelete extends BaseCommand {
 
     const { confirm } = await this.prompt(
       {
-        type: 'confirm',
+        type: 'text',
         name: 'confirm',
-        message: `Are you sure you want to delete database ${database} in the ${workspace} workspace?`,
+        message: `Are you sure you want to delete database ${database} in the ${workspace} workspace? Please type ${database} to confirm`,
         initial: false
       },
-      flags.force
+      flags.force ? database : undefined
```

Comment:
Non-blocking:

This line makes me think there might be a way to use `--force` to avoid entering the confirm phrase? 

If so I think adding 3x tests for this is might be a simple copy-paste job?